.github/workflows/vscode-icons.yml

```diff
@@ -19,19 +19,17 @@ jobs:
           submodules: 'recursive'
 
       - name: Update submodule
-        id: update_submodule
-        run: |
-          git submodule update --remote
-          cd vscode-icons/
-          tag=$(git describe --abbrev=0 --tags)
-          git reset --hard "$tag"
-          echo "latest_tag=$tag" >> "$GITHUB_OUTPUT"
+        id: submodules
+        uses: sgoudham/update-git-submodules@v2
+        with:
+          submodules: "vscode-icons"
+          strategy: "tag"
 
       - name: Create pull request
         uses: peter-evans/create-pull-request@v6
+        if: ${{ steps.submodules.outputs['vscode-icons--updated'] }}
```

Comment:
I don't think this `if` check is required since `peter-evans/create-pull-request` won't create a pull request with no changes, but I guess it is fine / more expliciit.

Reply:
Yeah you can technically remove the explicit `submodules` argument too since the repo only has one submodule at the moment.

That if check makes more sense when you're potentially updating multiple and you want separate PRs yeah.